server/engine/src/main/java/org/eclipse/lsp/cobol/core/model/extendedapi/InlinePerform.java

```diff
@@ -17,14 +17,18 @@
 import lombok.EqualsAndHashCode;
 import lombok.Value;
 
+import java.util.Optional;
+
 /** Data transport object of CF AST. InlinePerform type represents PERFORM UNTIL COBOL statement */
 @Value
 @EqualsAndHashCode(callSuper = true)
 public class InlinePerform extends CFASTNode {
-  PerformUntilType performUntilType;
+  String performUntilType;
```

Comment:
Add a test case please.

Reply:
How? Any suggestions?

Reply:
Whatever case that created the need for this change.

Reply:
I do not know how to do it. So if you know please clarify